pkg/server/core-db/migrations/1_entity.up.sql

```diff
@@ -0,0 +1,43 @@
+BEGIN;
```

Comment:
if we change things later, we'd add another file for the updates in this folder, right?